circular_linked_list/circular_linked_list.c

```diff
@@ -0,0 +1,53 @@
+#include<iostream>
+using namespace std;
+struct node
+{
+    int data;
+    node* next;
+};
+void show(node *head)
+{
+    node *current=head;
+    cout<<"The list is ";
+    do
+    {
+        cout<<current->data<<" ";
+        current=current->next;
+    }while(current!=head);
+    cout<<endl;
+}
+void insert(node **head,int n)
+{
+    node *link = new node;
+    link->data=n;
+    link->next=nullptr;
+    if(!*head){
+        link->next=link;
+        *head=link;
+    }
+    else
+    {
+        node *current = *head;
+        while(current->next!=*head)
+        {
+            current=current->next;
+        }
+        current->next=link;
+        link->next=*head;
+    }
+    show(*head);
+}
+
+int main()
+{
+    node *head=nullptr;
+    
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.space_checks`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpj00ez3q2/circular_linked_list/circular_linked_list.c
+++ b/tmp/tmpj00ez3q2/circular_linked_list/circular_linked_list.c
@@ -41,7 +41,7 @@
 int main()
 {
     node *head=nullptr;
-    
+
     insert(&head,4);
     insert(&head,2);
     insert(&head,8);
```

Reply:
Thanks for reviewing,will remove it now.